clickhouse-client/src/main/java/com/clickhouse/client/config/ClickHouseClientOption.java

```diff
@@ -343,6 +343,18 @@ public enum ClickHouseClientOption implements ClickHouseOption {
      * SSL key.
      */
     SSL_KEY("sslkey", "", "RSA key in PKCS#8 format.", true),
+    /**
+     * Key Store type.
+     */
+    KEY_STORE_TYPE("keystoretype", "", "Specifies the type or format of the keystore/truststore file used for SSL/TLS configuration, such as \"JKS\" (Java KeyStore) or \"PKCS12.\"", true),
```

Comment:
Can we use snake case for all 3 newly added options? Camel case is only used for JDBC specific options by the way.